### PR TITLE
switch to test-network

### PR DESCRIPTION
**Pull request by SamYuan1990**
Signed-off-by: SamYuan1990 <yy19902439@126.com>

### CODE REVIEW

File: integration-test.sh [hunk 1]
```diff
@@ -3,9 +3,13 @@ set -e
 
 DIR=$PWD
 
-curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash
-cd ./fabric-samples/first-network
-echo y | ./byfn.sh up
-cp -r crypto-config "$DIR" && cd "$DIR"
+curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash -s -- 2.1.1 1.4.7 0.4.20
```
Comment (guoger) [inline integration-test.sh:6]: why do need to specify version here (instead of default value in `bootstrap.sh`?

File: integration-test.sh [hunk 2]
```diff
@@ -3,9 +3,13 @@ set -e
...
+cd ./fabric-samples/test-network
+echo y |  ./network.sh up createChannel
+echo y |  ./network.sh deployCC
+cp -r organizations "$DIR" && cd "$DIR"
+mv organizations crypto-config
```
Comment (guoger) [inline integration-test.sh:11]: why don't we rename `crypto-config` in our own code to `organizations`?

File: integration-test.sh (hunk 1 above)
Comment (SamYuan1990) [reply integration-test.sh:6]: will try

File: integration-test.sh (hunk 2 above)
Comment (SamYuan1990) [reply integration-test.sh:11]: k

**Pull request merged**